inventory/allinone-ha.yaml

```diff
@@ -159,9 +159,42 @@ all:
           vars:
             service_state: stopped
 
+    # TODO(bzhaoopenstack): For forbid pollute online ENV, we use a test account
+    # to test post issue and change the github app webhook.
     keepalived:
       children:
         keepalived-master:
           hosts: allinone01
+          vars:
+            upstream_user: moo-ai
+            upstream_password: !vault |
+              $ANSIBLE_VAULT;1.1;AES256
+              31636661633365633364613732343738643135616137636566613536383833636138366338346337
+              6562623862363161316435323637633533656166343532620a353833366162393066353533343030
+              36666134313835616330373535356137653064656535313734306638376363663062653961646433
+              3466343735323338310a313963666339653934636630393437303239356236393234336562323065
+              6665
+            ori_env_role: "BACKUP"
         keepalived-slave:
           hosts: allinone02
+          vars:
+            nodepool_master_ip: "{{ hostvars[groups['nodepool-builder-master'][0]].ansible_host }}"
+            nodepool_slave_ip: "{{ hostvars[groups['nodepool-builder-slave'][0]].ansible_host }}"
+            zuul_master_ip: "{{ hostvars[groups['zuul-scheduler-master'][0]].ansible_host }}"
+            zuul_slave_ip: "{{ hostvars[groups['zuul-scheduler-slave'][0]].ansible_host }}"
```

Comment:
I am not sure if these variables can be the right values if define here, it is better to have a test. and I don't think it is better to move the above 4 variable from playbook to here.

Reply:
Already test in allinone with a sample test which is the same with style.. ;-). I think you and Huang core hit a disagreement. Ha. Eventhough I originally think the variable in playbook is much more clear than here.

Reply:
Mainly concern if the above 4 variable can have correct values, but looks like it can. I personally think the above 4 variables should be put in playbook. because these are not configurable values, but following variables are.  the user and password and app_name is related to the real deployment, So Ithink it is ok to put them here.

Reply:
Got it, thanks